ui-components/src/notifications/ApplicationStatus.stories.tsx

```diff
@@ -5,7 +5,6 @@ import advancedFormat from "dayjs/plugin/advancedFormat"
 import { ApplicationStatus } from "./ApplicationStatus"
 import { ApplicationStatusType } from "../global/ApplicationStatusType"
 import { t } from "../helpers/translator"
-import Archer from "../../__tests__/fixtures/archer.json"
```

Comment:
Changes here remove dynamic dates, which would produce a different screenshot every time. We also aren't using the listing object in this component anymore, so these are needed updates anyway.